numba_rvsdg/tests/test_fig4.py

```diff
@@ -30,3 +30,10 @@ def make_flow():
     flow = FlowInfo.from_bytecode(bc)
     bbmap = flow.build_basicblocks()
     return ByteFlow(bc=bc, bbmap=bbmap)
+
+def test_fig4():
+    render_flow(make_flow())
```

Comment:
Question: Using `render_flow` here will probably lead to the graph every time `pytest` suite is run locally. Can we  have a functional version of this which keeps rendering optional ?

Something like:
```python
if render_graph:
   render_flow(make_flow())
else:
    flow = make_flow()
    cflow = flow._join_returns()
    lflow = cflow._restructure_loop()
    bflow = lflow._restructure_branch()
```

Where `render_graph` is an environment boolean.

Reply:
I added a07a567 with the following comment:

```
When running the test via `pytest` nothing is displayed. When running
the file with `python` on the command line, the output is rendered.
Thus, the files can be used during testing but also to visualize the
results.
```